### PR TITLE
Makes shuttles init before atmos

### LINKED ISSUE #37614: Escape pods on metastation start out with no atmosphere
[Round ID]: # 87292

[Testmerges]: # I do not believe this to be caused by testmerges, this has happened on many rounds I've played on, all with different testmerges.

[Reproduction]: # Load up metastation, and head into an escape pod. This will create wind as the air from the hall is sucked into the escape pod chamber. Entering the escape pod will slowly start to kill you through atmos and lack of O2.

### PR DESCRIPTION
**Pull request by praisenarsie**
Fixes #37614 

HEY LISTEN!

### CODE REVIEW

**Comment by Cyberboss** [comment]
This can only be a bad idea

**Comment by Cyberboss** [comment]
My main concern is putting shuttles before lighting

**Comment by praisenarsie** [comment]
testmerge and instantly regret

i cant test this anyway but i'll put lighting before shuttles i suppose

File: code/__DEFINES/subsystems.dm [hunk 1]
```diff
@@ -66,15 +66,15 @@
...
+#define INIT_ORDER_SHUTTLE -1
+#define INIT_ORDER_AIR -2
 #define INIT_ORDER_MINIMAP -3
 #define INIT_ORDER_ASSETS -4
 #define INIT_ORDER_ICON_SMOOTHING -5
 #define INIT_ORDER_OVERLAY -6
 #define INIT_ORDER_XKEYSCORE -10
 #define INIT_ORDER_STICKY_BAN -10
 #define INIT_ORDER_LIGHTING -20
-#define INIT_ORDER_SHUTTLE -21
```
Comment (Jalleo) [inline code/__DEFINES/subsystems.dm]: Why didnt you just put air after here?

**Review by Jalleo — COMMENTED** [review]
TLDR

File: code/__DEFINES/subsystems.dm (hunk 1 above)
Comment (praisenarsie) [reply code/__DEFINES/subsystems.dm]: i am a very intelligent man but anyway does it really matter?

Comment (Jalleo) [reply code/__DEFINES/subsystems.dm]: Well then it wouldnt have a lighting issue if it did. Apart from that no idea.

Comment (praisenarsie) [reply code/__DEFINES/subsystems.dm]: meh

**Comment by praisenarsie** [comment]
what's efficiency

**Comment by praisenarsie** [comment]
Initialized Atmospherics subsystem within 98.6 seconds! Does it usually take this long? I forgot.

**Comment by cyclowns** [comment]
no it takes like 12 secs usually

**Comment by praisenarsie** [comment]
probably travis just being travis

**Comment by cyclowns** [comment]
image
not travis

**Comment by praisenarsie** [comment]
@Cyberboss any ideas?

**Comment by Cyberboss** [comment]
It's my day off

**Comment by Cyberboss** [comment]
Atmos initialization forces all inconsistent atmospheres to merge. That's
like the whole point

**Comment by optimumtact** [comment]
Yes but it can't do that accurately until all the subsystems have finished intialising because otherwise some the map components don't exist.
